### PR TITLE
Apple.com body

### PR DESCRIPTION
**Pull request by Tripple-A**
This is a pull request for my Apple.com clone page ready for review. Comments have been deleted and the Readme file has been updated further. Images have been put into the<a> tag. It has been validated and linted. It contains the stylelint  and stickler files if further checks are required.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <img class="second" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+    </a>
+</div>
+<div>
+                <a href="" class="bottom-title">Watch the “Your Verse” film and explore the stories</a>
+</div>
+       </div>
+    </div>
+    <div class="mid-section">
+        <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="360" height="200" alt="iPhone 5c. For the colorful.">
```
Comment (Ispirett) [inline index.html]: All 4 Images should be wrapped in `a` tags thanks

**Review by Ispirett — CHANGES_REQUESTED** [review]
Great job here with this porject @Tripple-A  :+1: 

There is a just a minor change below to checkout.

**Reminder please add more descriptive information to your readme.**
**Reminder please add a more descriptive commit  messages.**

File: index.html (hunk 1 above)
Comment (Tripple-A) [reply index.html]: Thanks... I have updated that. Please do take a look again.

Comment (Tripple-A) [reply index.html]: Also,can i please get your full name so that i can pick your name when filling the project completion form. Thanks.

Comment (Tripple-A) [reply index.html]: Also,my Read.me file is in my master branch.

**Comment by ebukaume** [comment]
Hi @Tripple-A :wave:

You are doing great :+1:, but there are some important requirements you slightly missed.

- [ ] The project should have a descriptive README.md
- [ ] Comments: It’s better to delete commented code. (If you need to review a previous version of your code, use git history.)

See [here](https://github.com/Tripple-A/Apple.com/blob/apple.com_body/css/top.css#L15)

You can refer to [this repo](https://github.com/othneildrew/Best-README-Template) for a clue on how to make your repo look more professional.

Note: Please remember to use a more descriptive PR title and description from now on.

**Please submit another code review request after fixing those issues.**

Happy coding :v:

**Comment by ebukaume** [comment]
Hi @Tripple-A,

Nice work so far :+1: . Let's make Stickler-CI run, as **it is a hard requirement for this project.**

- [ ] Check linter comments.
* Do not approve a project if it did not pass the linter check. 
* Make sure that students use config files copied from [config linters repo](https://github.com/microverseinc/linters-config#linters-config)

Refer to this [guide](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) if you need help debugging Stickler-CI

**Kindly submit another code review request after you have fixed those issues :v:**

**Review by bolah2009 — APPROVED** [review]
# It's a :+1: from me. 

### [The clone](https://rawcdn.githack.com/Tripple-A/Apple.com/3cd86ad624cd317440cc58478a873ddd7e35be66/index.html) looks good to me and uses CSS Backgrounds and Gradients effectively.

[OPTIONAL] Please fix the horizontal scroll, you do achieve this by preventing the use of absolute width sizes

#### :heavy_check_mark: This project is approved. :+1: 

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**